docker/README.md

```diff
@@ -0,0 +1,26 @@
+# Running your own Elastic Stack with Docker
+
+If you'd like to start Elastic locally, you can use the provided
+[docker-compose-elastic.yml](docker-compose-elastic.yml) file. This starts
+Elasticsearch, Kibana, and APM Server and only requires Docker installed.
+
+Note: If you haven't checked out this repository, all you need is one file:
+```bash
+wget https://raw.githubusercontent.com/elastic/elasticsearch-labs/refs/heads/main/docker/docker-compose-elastic.yml
```

Comment:
this link will work once merged. If you want to try before hand, switch "main" to "elastic-docker" (this branch)